index.html

```diff
@@ -275,28 +284,30 @@ <h2 class="individual__title">
                     <span class="individual__pager-link"></span>
                     <span class="individual__pager-link individual__pager-link--last"></span>
                 </div>
-                <div class="individual__pager-left">
+                <div class="slider-button slider-button--left">
                 </div>
-                <div class="individual__name individual__name--cherry individual__name--3">
+                <div class="individual__slider-container">
+                    <div class="individual__name individual__name--cherry individual__name--design">
```

Comment:
Please give the element `name` more suitable name like `slider-item`

Reply:
done